### ISSUE TITLE
New error using qProf: ValueError: max() arg is an empty sequence

### ISSUE THREAD

**Issue by lluisvicens** [mauroalberti/qProf#8]
Dear Mauro,
In the same context of my previous issue, another student get a new error trying to get a topographic profile based on a DEM and a vector line (route). The error information is attached as an image.

By the way, I've recorded a video about the process that the students should do to get the topographic profile. This is the exact process that takes me less than a minute to do, but some of them have to wait for dozens of minutes to do it... Is it a problem of RAM/Processor? 

Thanks in advance for your time and help.
Cheers,
Lluí
![valueerror](https://user-images.githubusercontent.com/1196051/35853804-dcc9d90c-0b2e-11e8-9e65-03111b96eea6.png)

**Comment by mauroalberti** [comment]
I reproduced the error, after some google search, by trying to plot a figure without any content (i.e., no topographic profile to draw), see (hopefully) in figure below.

![image](https://user-images.githubusercontent.com/1621998/35855970-ec5a78f8-0b34-11e8-9873-00b59ab657cb.png)

This matplotlib-related exception should have been fixed in matplotlib in July 2017, so possibly the matplotlibs that we are using are older.

I could handle better this particular case in the next release (not planned very soon).

With regards to the problem of some of your students, IF all the conditions between you and them are equal (same DEM, same trace, same QGIS software and project settings, same OS) apart from the hardware, the problems could be related to the memory amount available for the calculations..  But it should impact only when using large DEMs.

ciao,
mauro
